src/diffusers/models/modeling_utils.py

```diff
@@ -292,6 +295,153 @@ def disable_xformers_memory_efficient_attention(self):
         """
         self.set_use_memory_efficient_attention_xformers(False)
 
+    def add_adapter(self, adapter_config, adapter_name: str = "default") -> None:
+        r"""
+        Adds a fresh new adapter to the current model for training purpose. If no adapter name is passed, a default
+        name is assigned to the adapter to follow the convention of PEFT library (in PEFT we use "default" as the
+        default adapter name).
+
+        If you are not familiar with adapters and PEFT methods, we invite you to read more about them on the PEFT
+        official documentation: https://huggingface.co/docs/peft
+
+        Args:
+            adapter_config (`~peft.PeftConfig`):
```

Comment:
@stevhliu just double-checking for myself. Is this the right way of cross-referencing a library method in the docs?

Reply:
Hey! The library method should be bracketed like [`~peft.PeftConfig`] :)

Reply:
```suggestion
            adapter_config (`[~peft.PeftConfig]`):
```